news/2021-04-14-new-featured-artist-fiend.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "New Featured Artist: fiend"
+date: 2021-04-14 09:30:00 +0000
+---
+
+We're happy to welcome **fiend** into the Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/156/header.jpg)
+
+A trailblazer in his own musical niche, **fiend** is an Australian producer renowned for his iconic remix-based drum & bass tracks implementing themes from a diverse spread of genres like fierce rap, ambient trap, and simple EDM.
+
+We rounded up **5** tracks from **fiend**'s ever-popular *FEVER DREAM* album. While selection may be limited, each of these songs is an experience unlike any other, lending to more mapping opportunities than you'd expect from the short tracklist.
+
+Anyone who hasn't heard **fiend** is missing out. It's the honest truth. If you need an introduction, this cursor-tracking gameplay of the title track [*FEVER DREAM (feat. yzzyx)*](https://osu.ppy.sh/beatmapsets/1405913) mapped by [Niva](https://osu.ppy.sh/users/197805) is a perfect starting point, followed by any of the previews below:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/156/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### 2DF2
+
+Play through [this map of short the bonus track](https://osu.ppy.sh/beatmapsets/1288865) mapped by [0ppInOsu](https://osu.ppy.sh/users/12551840)!
```

Comment:
i think you meant to write `this map of the short bonus track` here tho (instead of `this map of short the bonus track`)